static/usage/v7/modal/can-dismiss/boolean/angular/example_component_html.md

```diff
@@ -20,8 +20,9 @@
         <ion-content>
           <p class="ion-padding-horizontal">You must accept the terms and conditions to close this modal.</p>
           <ion-item>
-            <ion-label class="ion-text-wrap" for="terms">Do you accept the terms and conditions?</ion-label>
-            <ion-checkbox id="terms" (ionChange)="onTermsChanged($event)" [checked]="canDismiss"></ion-checkbox>
+            <ion-checkbox id="terms" (ionChange)="onTermsChanged($event)" [checked]="canDismiss">
+              <ion-label class="ion-text-wrap" for="terms">Do you accept the terms and conditions?</ion-label>
```

Comment:
Can this just be a `span` with the class?

If this is a correct usage of `ion-label`, I'd argue that we should leave an input example similar to this in the label docs instead of removing that example entirely.

Reply:
Nah, this doesn't need to be an `ion-label`. I was just a little lazy here and copy-pasted the existing markup 😆